From 40f96f46434633aedd1820980a4c84097cfb7ea7 Mon Sep 17 00:00:00 2001
From: Georgy Yakovlev <168902+gyakovlev@users.noreply.github.com>
Date: Thu, 28 May 2020 00:22:19 +0000
Subject: [PATCH] etc/zfs/Makefile.am: set initconfdir

The initconfdir variable is not defined in etc/zfs/Makefile,
so the sed code does not perform the correct replacement.

Reviewed-by: Richard Yao <ryao@gentoo.org>
Reviewed-by: Brian Behlendorf <behlendorf1@llnl.gov>
Signed-off-by: Georgy Yakovlev <gyakovlev@gentoo.org>
Issue #10375
Closes #10376
---
 etc/zfs/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/etc/zfs/Makefile.am b/etc/zfs/Makefile.am
index 81567a4fa..5370f41bd 100644
--- a/etc/zfs/Makefile.am
+++ b/etc/zfs/Makefile.am
@@ -1,4 +1,5 @@
 pkgsysconfdir = $(sysconfdir)/zfs
+initconfdir = $(DEFAULT_INITCONF_DIR)
 
 pkgsysconf_DATA = \
 	vdev_id.conf.alias.example \
-- 
2.29.1

